<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="FirstPage" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="FirstNextButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="keyValue"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>or</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="GeneratePushButton">
          <property name="styleSheet">
           <string notr="true">#GeneratePushButton
{
	background-color: rgb(0, 170, 255);
}
</string>
          </property>
          <property name="text">
           <string>Generate Key</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="SecondPage" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>511</width>
       <height>301</height>
      </rect>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>471</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Type</string>
         </property>
         <widget class="QRadioButton" name="CrypRadioButt">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Encrypt</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="DecrypRadioButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Decrypt</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>471</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="inputText"/>
       </item>
       <item>
        <widget class="QTextEdit" name="outputText"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelOne">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTwo">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Crypto-text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HomeButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate new key</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExecuteButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>DecrypRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>labelOne</receiver>
   <slot>updateText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
